src/lib/server.ts

```diff
@@ -152,15 +152,16 @@ export class Client {
     } catch (err: unknown) {
       if (err instanceof BaseError) {
         this.logger.error("Message error", err);
-        return this.sendResultError(msg.messageId, err.errorCode);
+        const { errorCode, name, message, stack, ...args } = err;
```

Comment:
Is everything in `args` JSON serializable ?

Reply:
Yes, all of our errors use base types for arguments